### PR TITLE
Time dependent inputs for FHN, WC, Hopf

### PR DESCRIPTION
**Pull request by lenasal**
Include time-dependent inputs to the FHN model, the WC model, and the Hopf model.
- move "adjust_shape" function from aln time integration file to general file in utils
- fix bug in test_models
- add tests for stimulation of FHN, WC, Hopf

### CODE REVIEW

**Comment by caglorithm** [comment]
* Tests still failing. 
* PR contains changes in notebooks. 
* Changes in parameter names need to be reverted everywhere (also in tests).

**Comment by lenasal** [comment]
I've updated things, test are not failing anymore.

**Comment by lenasal** [comment]
changed index of inputs in time integration, this way it is more in line with the aln model (and also, I personally think it makes more sense)

**Comment by caglorithm** [comment]
Nice! Could you also remove your commits for the notebooks from the PR?

File: neurolib/models/hopf/loadDefaultParams.py [hunk 1]
```diff
@@ -77,8 +77,8 @@ def loadDefaultParams(Cmat=None, Dmat=None, seed=None):
     params.y_ou = np.zeros((params.N,))
 
     # values of the external inputs
-    params.x_ext = np.zeros((params.N,))
-    params.y_ext = np.zeros((params.N,))
```
Comment (caglorithm) [inline neurolib/models/hopf/loadDefaultParams.py:81]: Leaving these lines the way they are should work as well, right?

**Review by caglorithm — CHANGES_REQUESTED** [review]
Please see the one nit comment I've made!

**Comment by caglorithm** [comment]
LGTM. Merging after tests finish ✅.

**Pull request merged**